docs/views/examples/pass-data/index.html

```diff
@@ -81,6 +81,46 @@ <h3 class="govuk-heading-s">
 
       <h2 class="govuk-heading-m">Advanced features</h2>
 
+      <h3 class="govuk-heading-s">
+        Using the data in Nunjucks macros
+      </h3>
+
+      <p>Example using the 'checked' function in a radio component macro:</p>
```

Comment:
Suggest adding something like:

"The HTML example of this page is available on [Vehicle features](https://govuk-prototype-kit.herokuapp.com/docs/examples/pass-data/vehicle-features)."

This would help to signpost to the relevant page and call out the Nunjucks/HTML distinction.

Reply:
we already tell people how to use `checked` in html on this page, so I don't think we need to do this too

Reply:
`in a radio component macro`
should be
`in a checkbox macro` (do we need 'component'?)